### DISCUSSION TITLE
please list the events that OVOS emits

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#231] [Q&A] [answered]
dedicated docs here https://openvoiceos.github.io/ovos-technical-manual/skills_bus/

please list the emitted event

**Comment by JarbasAl** [comment, marked as answer]
a list of every event emited is being created here https://openvoiceos.github.io/message_spec/

it's not yet complete but its the best reference we currently have

**Reply by geetee24** [reply]
there is no data here.
https://openvoiceos.github.io/message_spec/

**Reply by JarbasAl** [reply]
![imagem](https://github.com/OpenVoiceOS/ovos-buildroot/assets/33701864/0cd30e2d-8bda-48b2-b849-d864785b4011)

**Reply by geetee24** [reply]
what is that URL please to the main section.

**Reply by geetee24** [reply]
i googles the www and cant find the url to OVOS mess docs.
